samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java

```diff
@@ -0,0 +1,105 @@
+    String alertPolicyId = "alert-policy-id";
+    // i.e projects/[PROJECT_ID_OR_NUMBER]/alertPolicies/[ALERT_POLICY_ID]
+    enableDisableAlertPolicy(alertPolicyId, false, displayName);
+  }
+
+  public static void enableDisableAlertPolicy(
+      String alertPolicyId, boolean status, String displayName) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
+          "metric.type=\"compute.googleapis.com/instance/"
+              + "cpu/utilization\" AND resource.type=\"gce_instance\"";
+
+      // Build Duration
+      Duration aggregationDuration = Duration.newBuilder().setSeconds(60).build();
+
+      // Build Aggregation
+      Aggregation aggregation =
+          Aggregation.newBuilder()
+              .setAlignmentPeriod(aggregationDuration)
+              .setCrossSeriesReducer(Aggregation.Reducer.REDUCE_MEAN)
+              .setPerSeriesAligner(Aggregation.Aligner.ALIGN_MAX)
+              .build();
+
+      // Build MetricThreshold
+      AlertPolicy.Condition.MetricThreshold metricThreshold =
+          AlertPolicy.Condition.MetricThreshold.newBuilder()
+              .setComparison(ComparisonType.COMPARISON_GT)
+              .addAggregations(aggregation)
+              .setFilter(metricFilter)
+              .setDuration(aggregationDuration)
+              .build();
+
+      // Construct condition
+      AlertPolicy.Condition condition =
+          AlertPolicy.Condition.newBuilder()
+              .setDisplayName(displayName)
+              .setConditionThreshold(metricThreshold)
+              .build();
+
+      // Create alert policy for update
+      AlertPolicy alertPolicy =
+          AlertPolicy.newBuilder()
+              .setName(alertPolicyId)
+              .setDisplayName(displayName)
+              .addConditions(condition)
+              .setCombiner(AlertPolicy.ConditionCombinerType.AND)
+              .setEnabled(
+                  BoolValue.newBuilder()
+                      .setValue((status == true) ? Boolean.TRUE : Boolean.FALSE)
+                      .build())
+              .build();
+
+      // Create update alert policy request
+      UpdateAlertPolicyRequest updateAlertPolicyRequest =
+          UpdateAlertPolicyRequest.newBuilder().setAlertPolicy(alertPolicy).build();
+
+      // Process update policy request
+      AlertPolicy updatedAlertPolicy =
+          alertPolicyServiceClient.updateAlertPolicy(updateAlertPolicyRequest);
+      System.out.print(
+          "alert policy enable disable status:" + updatedAlertPolicy.getEnabled() + "\n");
+    } catch (ApiException ex) {
+      System.out.print("alert policy not found:" + ex.toString());
+    }
```

Comment:
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

Reply:
Done